rnbdsh@negateWindow/files/rnbdsh@negateWindow/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "external-configuration-app": "extension.js", 
```

Comment:
Hello, @rnbdsh.

You should remove `external-configuration-app` line. It's a remnant of the extension you based it on and it's not used by your extension.